temprl/wrapper.py

```diff
@@ -90,6 +90,30 @@ def step(self, symbol: Interpretation) -> Tuple[State, float]:
         """
         return self._simulator.step(symbol)
 
+    def current_dfa_state(self) -> State:
+        """Get the current DFA state."""
+        return self._simulator._current_state
+
+
+class StepController:
```

Comment:
Would be great to have docstrings, e.g.:

```suggestion
class StepController:
    """A class that allows to control the steps to be done by the temporal goals."""
```

Reply:
Added in [39b42fa](https://github.com/whitemech/temprl/pull/44/commits/39b42faeb7322572c92c6842be8e62425fb5fadd).